R/settings_utils.R

```diff
@@ -68,30 +68,24 @@ find_config <- function(filename) {
     dirname(filename)
   }
 
-  ## check for a file in the current directory
-  linter_config <- file.path(path, linter_file)
-  if (isTRUE(file.exists(linter_config))) {
-    return(linter_config)
-  }
-
-  ## next check for a file higher directories
-  linter_config <- find_config2(path)
-  if (isTRUE(file.exists(linter_config))) {
-    return(linter_config)
-  }
-
-  ## next check for a file in the user directory
-  # cf: rstudio@bc9b6a5 SessionRSConnect.R#L32
-  home_dir <- Sys.getenv("HOME", unset = "~")
-  linter_config <- file.path(home_dir, linter_file)
-  if (isTRUE(file.exists(linter_config))) {
-    return(linter_config)
-  }
-
-  ## next check for a file in the .github/linters directory
-  linter_config <- file.path(path, ".github/linters", linter_file)
-  if (isTRUE(file.exists(linter_config))) {
-    return(linter_config)
+  # List possible .lintr file locations
+  file_locations <- list(
```

Comment:
use `c()` instead of `list()` so that `file_locations` is a character vector, not a list.

Reply:
Updated, thanks!